packages/xo-web/src/xo-app/proxies/update-appliance-settings.js

```diff
@@ -0,0 +1,74 @@
+import _, { messages } from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { form } from 'modal'
+import { generateId } from 'reaclette-utils'
+import { injectIntl } from 'react-intl'
+import { provideState, injectState } from 'reaclette'
+import { updateProxyAppliance } from 'xo'
+
+const Modal = decorate([
```

Comment:
`updateHttpProxyModal` ?

Reply:
This modal is dedicated to the appliance settings so `UpdateApplianceSettingsModal` ?